pkg/BUILD

```diff
@@ -42,11 +42,22 @@ py_binary(
 )
 
 
-# NOTE: This works for PY2 and PY3.
-# TODO: Figure out how to use "whatever version" we have on the machine.
 py_binary(
     name = "make_deb",
     srcs = ["make_deb.py"],
+    python_version = "PY3",
+    srcs_version = "PY3",
```

Comment:
I'd actually leave out `srcs_version` entirely and let it default to `PY2AND3`. The source code is compatible with both versions, after all. It's only the target that says to use one or the other.

If this were a `py_library` target, `PY2AND3` would be necessary in order for the target to be usable as a dependency in binaries of both versions. It doesn't matter for `py_binary` except for documentation / clarity.

Actually, setting it to the default `PY2AND3` value explicitly could even be helpful, in that it replaces the "This works for both PY2 and PY3" comment. Same principle for the `make_deb_py2` target.

Reply:
Done.